docs/tutorials/e2e/prerequisites/prerequisites.md

```diff
@@ -137,7 +137,8 @@ export NO_PROXY="localhost,127.0.0.1,::1"
 
 #### http (80)
 
-The port http (80) should not be used, but in case it will, you can apply the above hints just by replacing https by http.
+The port http (80) should not be used, but it will, you can apply the above hints just by replacing https by http.
```

Comment:
For me this sentence makes not so much sense, to be honest.
- Why are we saying, it should not been used when it will be used?
- Which above hints? Directly above we find the proxy configuration. Maybe it makes sense to put a link here.
- Did you missed a `.`? The sentence makes more sense if we replace the `,` after `but it will` with an `.`: `The port http (80) should not be used, but it will. You can apply the above hints just by replacing https by http.`

Reply:
Maybe we just remove, that the port shouldn't be used, and write:

`For port 80, which is currently used, please apply the above hints just by replacing https by http.`

Reply:
@florianrusch-zf agree. And maybe additional to this, I think we should avoid links like in line 133
```bash
export https_proxy=http://arena2036-proxy.rus.uni-stuttgart.de:8080
```
-> Because the documentation should be more generic. We had the same discussions with previous named links like the daps (long time ago :)) aso. We also deleted them, and I think in this cold we should do the same.

Reply:
then please do not delezt , instead replace by:

`export https_proxy=http://<hostnme>:8080`

Reply:
Please just change as you like
Thanks
Johannes

Reply:
Dedicated proxy was removed. Suggestion for sentence and above hint was added